doc/nrf/known_issues.rst

```diff
@@ -220,6 +220,20 @@ Assert risk after performing a DLE procedure
 Other issues
 ------------
 
+.. rst-class:: v1-4-0 v1-3-2 v1-3-1 v1-3-0
+
+NCSDK-6844: Security procedure failure can terminate GATT client request
+  Security procedure terminates the current GATT client request in progress when this request was not the reason to initiate the security procedure.
+  If a new GATT client request is queued then this can now potentially cause a GATT transaction violation and could fail as well.
+
+  **Workaround:** Don't initiate security procedure in parallel with GATT client requests.
+
+.. rst-class:: v1-4-0
```

Comment:
NCSDK-6872. Is that only for NCS 1.4.0?

Reply:
Removed it, but don't know which releases are affected yet.